### PR TITLE
Docs update:

### PR DESCRIPTION
**Pull request by seldo**
* New introduction and installation tweaks
* New use-case based starter tutorials
* Adding a new Guides section with Agents to start
* Miscellaneous docs bug fixes

### CODE REVIEW

File: apps/docs/docs/examples/other_llms.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import CodeBlock from "@theme/CodeBlock";
+import CodeSource from "!raw-loader!../../../../examples/mistral";
+
+# Using other LLM APIs
```
Comment (logan-markewich) [inline apps/docs/docs/examples/other_llms.mdx:4]: FYI one super common question I see on discord is how to use azure modrld

Not sure if that's in the examples folder or should have a dedicated spot in the docs or not

Comment (seldo) [reply apps/docs/docs/examples/other_llms.mdx:4]: Good feedback! I will include it in the next rev somewhere, this believe it or not is just a WIP.

File: apps/docs/docs/guides/agents/2_create_agent.mdx [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+```javascript
+Settings.callbackManager.on("llm-tool-call", (event) => {
+  console.log(event.detail.payload);
+});
+Settings.callbackManager.on("llm-tool-result", (event) => {
+  console.log(event.detail.payload);
+});
+```
+
+### Create a function
```
Comment (logan-markewich) [inline apps/docs/docs/guides/agents/2_create_agent.mdx:60]: Should we link Alex's tool library?

Comment (seldo) [reply apps/docs/docs/guides/agents/2_create_agent.mdx:60]: Another good idea! I'll need to learn more about how it works.

File: apps/docs/docs/getting_started/starter_tutorial/chatbot.mdx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+sidebar_position: 2
+---
+
+# Chatbot tutorial
+
+Once you've mastered basic [retrieval-augment generation](retrieval_augmented_generation) you may want to create an interface to chat with your data. You can do this step-by-step, but we recommend getting started quickly using `create-llama`.
+
+## Using create-llama
+
+`create-llama` is a powerful but easy to use command-line tool that generates a working, full-stack web application that allows you to chat with your data. You can learn more about it on [the `create-llama` README page](https://www.npmjs.com/package/create-llama).
```
Comment (marcusschiesser) [inline apps/docs/docs/getting_started/starter_tutorial/chatbot.mdx:11]: create-llama also supports adding agent tools (Wikipedia, Weather, Web search) to the chat. If tools are added AND the user selects custom data (create-llama calls that datasources), then a QueryEngineTool is added.

**Comment by yisding** [comment]
Good comments. Merging for now. Let's put the updates in a separate PR.

**Pull request merged**